### PR TITLE
Show events on components and replicas

### PR DESCRIPTION
**Pull request by satr**

### CODE REVIEW

File: src/components/replica/index.tsx [hunk 1]
```diff
@@ -55,7 +55,7 @@ const ReplicaDuration: FunctionComponent<{ created: Date; ended?: Date }> = ({
   );
 };
 
-const ContainerDuration: FunctionComponent<{ started: Date; ended?: Date }> = ({
+const ContainerDuration: FunctionComponent<{ started: Date; ended: Date }> = ({
```
Comment (Richard87) [inline src/components/replica/index.tsx:58]: This is still used as possible undefined?
```suggestion
const ContainerDuration: FunctionComponent<{ started: Date; ended?: Date }> = ({
```

File: src/components/replica/index.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ interface ReplicaElements {
   state?: React.JSX.Element;
 }
 
-const ReplicaDuration: FunctionComponent<{ created: Date; ended?: Date }> = ({
+const ReplicaDuration: FunctionComponent<{ created: Date; ended: Date }> = ({
```
Comment (Richard87) [inline src/components/replica/index.tsx:25]: ended is possibly undefined
```suggestion
const ReplicaDuration: FunctionComponent<{ created: Date; ended?: Date }> = ({
```

**Review by Richard87 — APPROVED** [review]
Looks good, a linting error, and I miss my `?` :D

**Comment by satr** [comment]
> Looks good, a linting error, and I miss my `?` :D

Fixed

**Review by Richard87 — APPROVED** [review]
Godkjent

**Pull request merged**